### PR TITLE
Add attribute type="tel" to input tag

### PR DESCRIPTION
**Pull request by dougdomeny**
The "tel" type informs touch devices to display the telephone number keyboard in addition to other specialized behaviors--like initiating a phone call.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-input-type-tel](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-input-type-tel)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/phone-field",
-  "version": "4.5.0",
+  "version": "4.6.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: We update the package version in a separate PR; do you mind removing this and rerunning `npm install` (to remove the package-lock.json change)?

Comment (dougdomeny) [reply package.json]: Done.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-input-type-tel](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-input-type-tel)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**